static/docs/commands-reference/remote:modify.md

```diff
@@ -89,7 +89,7 @@ these settings, you could use the following options:
     $ dvc remote modify myremote use_ssl false
   ```
 
-* `listobjects` - whether or not to use `list_objects`. By default, `list_objects_v2` is used. Useful for ceph and other s3 emulators.
```

Comment:
Sorry, I think there is some misunderstanding. I didn't mean to keep the comment length under 80. Actually the initial text you put together is great! And it's good to know (being able to search for ceph, for example)! I just wanted to wrap the lines at 80. Multiple lines are absolutely fine. Thank you for PR!

Reply:
@shcheklein  should be fixed now.  Let me know if other issues.